dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qt5-build
+
+LANGS="ar da de en es fa fi fr hi it ja ko nb pl pt ru sv zh_CN zh_TW"
+
+DESCRIPTION="Virtual keyboard plugin for the Qt5 framework"
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~x86"
+fi
+
+IUSE="+hunspell"
```

Comment:
probably just "spell"... hunspell/aspell should be used only to distinguish among different spell-checking providers when the package supports more than one.